src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/MovementService.kt

```diff
@@ -1,28 +1,10 @@
 package uk.gov.justice.digital.hmpps.welcometoprison.model
 
-import com.github.javafaker.Faker
 import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.welcometoprison.model.basm.BasmService
 import java.time.LocalDate
-import java.time.ZoneId
-import java.util.Random
 
 @Service
-class MovementService(val faker: Faker = Faker()) {
```

Comment:
was half tempted to leave this in and add the new api on a separate route to not disturb testing the frontend